setup.py

```diff
@@ -13,63 +13,64 @@ class PyTest(TestCommand):
 
     def finalize_options(self):
         TestCommand.finalize_options(self)
-        # we don't run integration tests which need an actual Sonos device
-        self.test_args = ['-m', 'not integration']
+        # We don't run integration tests which need an actual Sonos device
+        self.test_args = ["-m", "not integration"]
         self.test_suite = True
 
     def run_tests(self):
-        # import here, cause outside the eggs aren't loaded
+        # Import here, because outside the eggs aren't loaded
         import pytest
+
         errno = pytest.main(self.test_args)
         sys.exit(errno)
 
 
-src = io.open('soco/__init__.py', encoding='utf-8').read()
-metadata = dict(re.findall("__([a-z]+)__ = \"([^\"]+)\"", src))
+src = io.open("soco/__init__.py", encoding="utf-8").read()
+metadata = dict(re.findall('__([a-z]+)__ = "([^"]+)"', src))
 docstrings = re.findall('"""(.*?)"""', src, re.MULTILINE | re.DOTALL)
 
-NAME = 'soco'
+NAME = "soco"
 
 PACKAGES = (
-    'soco',
-    'soco.plugins',
-    'soco.music_services',
+    "soco",
+    "soco.plugins",
+    "soco.music_services",
 )
 
-TEST_REQUIREMENTS = list(open('requirements-dev.txt'))
-AUTHOR_EMAIL = metadata['author']
-VERSION = metadata['version']
-WEBSITE = metadata['website']
-LICENSE = metadata['license']
+TEST_REQUIREMENTS = list(open("requirements-dev.txt"))
+AUTHOR_EMAIL = metadata["author"]
+VERSION = metadata["version"]
+WEBSITE = metadata["website"]
+LICENSE = metadata["license"]
 DESCRIPTION = docstrings[0]
 
 CLASSIFIERS = [
-    'Development Status :: 3 - Alpha',
-    'Intended Audience :: Developers',
-    'License :: OSI Approved :: MIT License',
-    'Operating System :: OS Independent',
-    'Programming Language :: Python :: 2',
-    'Programming Language :: Python :: 2.7',
-    'Programming Language :: Python :: 3',
-    'Programming Language :: Python :: 3.5',
-    'Programming Language :: Python :: 3.6',
-    'Programming Language :: Python :: 3.7',
-    'Programming Language :: Python :: 3.8',
-    'Programming Language :: Python :: Implementation :: CPython',
-    'Programming Language :: Python :: Implementation :: PyPy',
-    'Topic :: Home Automation',
-    'Topic :: Multimedia :: Sound/Audio',
-    'Topic :: Multimedia :: Sound/Audio :: Players',
-    'Topic :: Software Development :: Libraries :: Python Modules',
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3.5",
+    "Programming Language :: Python :: 3.6",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
```

Comment:
Optionally make it more explicit:
```suggestion
    "Programming Language :: Python :: 3.9",
    "Programming Language :: Python :: 3 :: Only",
```

Reply:
Thanks, done.